doc/index.md

```diff
@@ -3,27 +3,7 @@ layout: doc
 title: Documentation
 ---
 
-<p id="badges">
-  <a href="https://www.nuget.org/packages/Npgsql/">
-    <img src="https://img.shields.io/nuget/v/Npgsql.svg?label=stable&amp;style=plastic;maxAge=600" />
-  </a>
-
-  <a href="https://www.myget.org/gallery/npgsql-unstable">
-    <img src="https://img.shields.io/myget/npgsql-unstable/vpre/npgsql.svg?label=unstable&amp;style=plastic;maxAge=600" />
-  </a>
-
-  <a href="https://ci.appveyor.com/project/roji/npgsql">
-    <img src="https://img.shields.io/appveyor/ci/roji/npgsql/dev.svg?label=AppVeyor" />
-  </a>
-
-  <a href="https://travis-ci.org/npgsql/npgsql" >
-    <img src="https://img.shields.io/travis/npgsql/npgsql.svg?label=Travis" />
-  </a>
-
-  <a href="https://gitter.im/npgsql/npgsql">
-    <img src="https://img.shields.io/badge/gitter-join%chat-brightgreen.svg?style=plastic;maxAge=600" />
-  </a>
-</p>
+[![stable](https://img.shields.io/nuget/v/Npgsql.svg?label=stable)](https://www.nuget.org/packages/Npgsql/) [![unstable](https://img.shields.io/myget/npgsql-unstable/v/npgsql.svg?label=unstable)](https://www.myget.org/feed/npgsql-unstable/package/nuget/Npgsql) [![appveyor](https://img.shields.io/appveyor/ci/roji/npgsql/dev.svg?label=appveyor)](https://ci.appveyor.com/project/roji/npgsql) [![travis](https://img.shields.io/travis/npgsql/npgsql.svg?label=travis)](https://travis-ci.org/npgsql/npgsql) [![gitter](https://img.shields.io/badge/gitter-join%20chat-brightgreen.svg)](https://gitter.im/npgsql/npgsql)
```

Comment:
How about adding `[![upcoming_minor](https://img.shields.io/myget/npgsql/v/npgsql.svg?label=upcoming%20minor)](https://www.myget.org/feed/npgsql/package/nuget/Npgsql) ` here?

Reply:
That sounds good. For brevity, what about `next minor` instead of `upcoming minor`? (Want to avoid a line wrap.)

Reply:
> For brevity, what about next minor instead of upcoming minor? (Want to avoid a line wrap.)

However you like it. I don't care that much.